src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -349,6 +350,32 @@ public function customersHavePlacedOrdersForTotalOf($numberOfCustomers, $numberO
     }
 
     /**
+     * @Given the customer canceled this order
+     */
+    public function theCustomerCanceledThisOrder()
+    {
+        /** @var OrderInterface $order */
+        $order = $this->sharedStorage->get('order');
+
+        $order->setState(OrderInterface::STATE_CANCELLED);
```

Comment:
Shouldn't we trigger state machine "cancel" transition rather than just set a state?